base/src/main/java/com/thoughtworks/go/util/DateUtils.java

```diff
@@ -40,7 +40,7 @@ public static String formatIso8601ForCCTray(Date date) {
         if (date == null) {
             return null;
         }
-        return new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss").format(date);
+        return formatter.withZoneUTC().print(date.getTime());
```

Comment:
Seems like a reasonable change. Can you extract `formatter.withZoneUTC()` into another  constant to avoid additional overhead each time?